Xamarin.Forms.Core/Markup/BindableObjectExtensions.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System;
+
+namespace Xamarin.Forms.Markup
+{
+	public static class BindableObjectExtensions
+	{
+		const string bindingContextPath = Binding.SelfPath;
+
+		/// <summary>Bind to a specified property</summary>
+		public static TBindable Bind<TBindable>(
+			this TBindable bindable,
+			BindableProperty targetProperty,
+			string path = bindingContextPath,
+			BindingMode mode = BindingMode.Default,
+			IValueConverter converter = null,
+			object converterParameter = null,
+			string stringFormat = null,
+			object source = null,
+			object targetNullValue = null,
+			object fallbackValue = null
+		) where TBindable : BindableObject
+		{
+			bindable.SetBinding(
+				targetProperty, 
+				new Binding(path, mode, converter, converterParameter, stringFormat, source)
+				{
+					TargetNullValue = targetNullValue,
+					FallbackValue = fallbackValue
+				});
+			return bindable;
+		}
+
+		/// <summary>Bind to a specified property with inline conversion</summary>
+		public static TBindable Bind<TBindable, TSource, TDest>(
```

Comment:
it looks like a missed opportunity to have TSource and TDest and still generate a conventional binding

Reply:
Could you clarify? It does generate a conventional binding, this overload is shorthand for creating  a converter and passing that to the standard binding.